Parsers/ASimAuditEvent/Parsers/vimAuditEventCiscoMeraki.yaml

```diff
@@ -0,0 +1,260 @@
+Parser:
+  Title: Audit Event ASIM parser for Cisco Meraki
+  Version: '0.1'
+  LastUpdated: Jun 20 2023
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: Cisco Meraki Documentation
+  Link: https://documentation.meraki.com/
+Description: |
+  This ASIM parser supports normalizing Cisco Meraki logs to the ASIM Audit Event normalized schema. Cisco Meraki events are generated from network activity and security events from Meraki devices such as firewalls, switches, and access points. These logs are captured through the Cisco Meraki Sentinel connector which uses a Linux agent to collect logs in Syslog format.
+ParserName: vimAuditEventCiscoMeraki
+EquivalentBuiltInParser: _Im_AuditEvent_CiscoMeraki
+ParserParams:
```

Comment:
we are missing few audit filters. please include them too. If we don't have values in those fields, we can check for fitlers containing null or '*'
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-audit#filtering-parser-parameters

Reply:
Done